TACC/Core-Portal
# task/FP-889: Add searchbar to Public Data view.

## Overview: ##

## Related Jira tickets: ##

* [FP-889](https://jira.tacc.utexas.edu/browse/FP-889)

## Summary of Changes: ##
Adds a search bar to the "true public" data view.
## Testing Steps: ##
1. Navigate to https://cep.dev/public-data/tapis/public/cep.storage.public/
2. Enter a search query and confirm that results appear.

## UI Photos:

## Notes: ##
